org.lflang/src/org/lflang/generator/ReactionInstanceGraph.java

```diff
@@ -94,6 +94,14 @@ public void rebuild() {
         }
     }
     
+    /*
+     * Get an array of non-negative integers representing the maximum number of reactions 
+     * per each level, where levels are indices of the array.
+     */
+    public Integer[] getMaxNumOfReactionPerLevel() {
+        return maxNumOfReactionPerLevel.toArray(new Integer[0]);
```

Comment:
I find the terminology used here a bit confusing; this is not really about the maximum, but rather the total number of reactions per level.

Reply:
You're right.